cmd/zfs/zfs_main.c

```diff
@@ -4417,13 +4488,17 @@ zfs_do_send(int argc, char **argv)
 		{"backup",	no_argument,		NULL, 'b'},
 		{"holds",	no_argument,		NULL, 'h'},
 		{"saved",	no_argument,		NULL, 'S'},
+		{"exclude",	required_argument,	NULL, 'X'},
 		{0, 0, 0, 0}
 	};
 
 	/* check options */
-	while ((c = getopt_long(argc, argv, ":i:I:RsDpvnPLeht:cwbd:S",
+	while ((c = getopt_long(argc, argv, ":i:I:RsDpvnPLeht:cwbd:SX:",
```

Comment:
GitHub doesn't let you comment on untouched lines, but: you also need to add `[-X dataset[,dataset]...]` to (a) `do_zfs_send()` and (b) the usage strings.